mozilla-mobile/fenix/app/src/main/res/values-azb/strings.xml

```diff
@@ -291,7 +291,7 @@
   <!-- Browser menu label to sign back in to sync on the device when the user's account needs to be reauthenticated -->
   <string name="browser_menu_sign_back_in_to_sync">دؤنگل اوچون گئنه گیرین</string>
   <!-- Browser menu caption label for the "Sign back in to sync" browser menu item described in `browser_menu_sign_back_in_to_sync` when there is an error in syncing -->
-  <string name="browser_menu_syncing_paused_caption">\u0020دؤنگل دایاندیریلدی</string>
+  <string name="browser_menu_syncing_paused_caption">دؤنگل دایاندیریلدی</string>
```

Comment:
Is GitHub diff just being weird with RTL here and in the next one? I'm guessing it think that the space is at the end.

Reply:
This is GitHub's web UI being weird.